bogota-july-2018/Dominik Ortiz Galop/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
Just to be sure, you know what these meta means ?

Reply:
Yep 
<meta name="viewport" content="width=device-width, initial-scale=1.0"> it's a html5 tag that let me control the viewport it's use in RWD (https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag ) and <meta http-equiv="X-UA-Compatible" content="ie=edge"> choose what version of Internet Explorer the page should be rendered as. but visual studio code always add this 2 tags to my code and i forgot to remove for this project

Reply:
ohh and <meta charset="UTF-8"> to let know the browser which character set to use. in this case UTF-8"